packages/editor/src/store/selectors.js

```diff
@@ -1730,7 +1730,7 @@ export const __experimentalGetDefaultTemplateType = createSelector(
 			) ?? EMPTY_OBJECT
 		);
 	},
-	( state, slug ) => [ __experimentalGetDefaultTemplateTypes( state ), slug ]
+	( state ) => [ __experimentalGetDefaultTemplateTypes( state ) ]
```

Comment:
`slug` is a redundant dependant, because it's a selector parameter. When `createSelector` stores the memoized values, they are automatically keyed by the parameters.

Having `slug` there can also defeat the memoization, because there is a "reset the cache when dependants change" logic.

Reply:
Nice find 👍